core/relay/scripts/convert-compose.sh

```diff
@@ -24,17 +24,17 @@ uncomment() {
 
 if [ $1 -eq 2 ]
 then
-    uncomment 66,68 docker-compose.yaml
-    uncomment 74,75 docker-compose.yaml
-    uncomment 84 docker-compose.yaml
-    uncomment 105 docker-compose.yaml
-    comment 104 docker-compose.yaml
+    uncomment 69,73 docker-compose.yaml
```

Comment:
As per above comment, may be add `uncomment 59,61 docker-compose.yaml`.

Reply:
Done.